frontend/model/captureLogs.js

```diff
@@ -174,7 +186,7 @@ function setAppLogsFilter (filter: Array<string>) {
 window.addEventListener('beforeunload', event => sbp('appLogs/save'))
 
 sbp('sbp/selectors/register', {
-  'appLogs/download' (elLink) { downloadLogs(elLink) },
+  'appLogs/downloadOrShare' (...args) { downloadOrShareLogs(...args) },
```

Comment:
Could you change this to:

```js
'appLogs/downloadOrShare': downloadOrShareLogs,
```

It's simpler and also preserves the return value of `downloadOrShareLogs`.

A similar modification can also be done for `'appLogs/pauseCapture'` and `'appLogs/startCapture'` below.